server/api.go

```diff
@@ -83,6 +84,38 @@ func (p *Plugin) PublishStatusChanged(w http.ResponseWriter, r *http.Request) {
 	writeStatusOK(w)
 }
 
+func (p *Plugin) GetStatusByEmail(w http.ResponseWriter, r *http.Request) {
+	params := mux.Vars(r)
+	email := params["email"]
```

Comment:
constant for this param?

Reply:
My bad, some changes were pushed which were not supposed to be. Updated now.